tensorhive/models/auth/RevokedTokenModel.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+from tensorhive.database import Base, db_session
+
+
+class RevokedTokenModel(Base):
+    __tablename__ = 'revoked_tokens'
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    jti = Column(String(120), nullable=False)
+
+    def __repr__(self):
+        return '<RevokedToken: id={self.jti}, jti={self.jti}>'
```

Comment:
This won't work. It will explicitly print 'id={self.jti}' without string interpolation.
Use `format()` with named arguments.

Reply:
Already fixed.